src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -316,8 +316,8 @@ public function request(string $method, string $url, array $options = []): Respo
         }
 
         foreach ($curlopts as $opt => $value) {
-            if (\CURLOPT_INFILESIZE === $opt && $value >= 1 << 31) {
-                $opt = 115; // 115 === CURLOPT_INFILESIZE_LARGE, but it's not defined in PHP
+            if (\PHP_INT_SIZE === 8 && \defined('CURLOPT_INFILESIZE_LARGE') && \CURLOPT_INFILESIZE === $opt && $value >= 1 << 31) {
```

Comment:
But before the assignment also happened when the constant is not defined, right? Now it need to be defined 🤔

Reply:
The cURL extension doesn't support option it doesn't know: https://github.com/php/php-src/blob/790286ab6c8954193bbbca778749550a51aceb97/ext/curl/interface.c#L2358